modules/Router.js

```diff
@@ -57,6 +62,26 @@ const Router = React.createClass({
   },
 
   componentWillMount() {
+    const { history, transitionManager, router } = this.createRouterObjects()
+
```

Comment:
Can you move the parseQueryString/stringifyQuery check up here? It clouds up the purpose of `createRouterObjects` a bit and seems more sensible to leave as a mounting check. If for some reason we need to recreate the router objects (hot reloading, most likely), it'll warn again.